terraform/variables/demo-gcp.tfvars

```diff
@@ -4,10 +4,13 @@ gcp_project                = "prio-bringup-290620"
 machine_type               = "e2-small"
 peer_share_processor_names = ["test-pha-1", "test-pha-2"]
 aws_region                 = "us-west-1"
-# Graciously donated by jrenken
-manifest_domain = "portcull.is"
+manifest_domain            = "isrg-prio.org"
+managed_dns_zone = {
+  name        = "manifests"
+  gcp_project = "prio-bringup-290620"
```

Comment:
Would it make sense for this (specifically gcp_project) to be a top level setting? Are we going to need to use gcp_project elsewhere?

Reply:
There is a top level variable for `gcp_project`. The rationale here, which I called out in comment in `manifest.tf`, is that the Google managed DNS zone object might not be in the same GCP project as the ENV you are deploying. Specifically, you created https://console.cloud.google.com/net-services/dns/zones/certificates/details?project=prio-bringup-290620 in the `prio-bringup-290620` project. As I understand it, we can only have one managed zone globally for `manifests.isrg-prio.org.`, so if @jsha wants to issue certs from his deployment in the project he started, or if we spin up a new project for the eventual `prod-us` deployment, we need those other envs to be able to create DNS records in a managed zone in a different project, which is why I provide that here, separately from the top-level `gcp_project` value. I haven't yet tested creating DNS records across GCP projects in the manner I describe, though.